### PR TITLE
Added minor styling changes

### PR DESCRIPTION
**Pull request by chloejonessoftwire**
all tests passed, fixed header/footer overlaps, added a border to boxes and changed one font size on rover selection page

### CODE REVIEW

File: src/RoverSelection/RoverSelection.scss [hunk 1]
```diff
@@ -85,8 +87,9 @@ table, th, td {
 .toppagebreak{
     padding-top: 80px;
 }
-a{font-size: large;
-    color:$lighttextcolor;
+
+a{font-size: larger;
```
Comment (ChristianMcc) [inline src/RoverSelection/RoverSelection.scss]: Formatting here

File: src/Curiosity/curiosity.tsx [hunk 2]
```diff
@@ -48,18 +48,26 @@ function Curiosity() {
 
     return (
         <div>
+
+            <div className='toppagebreak'></div>
```
Comment (ChristianMcc) [inline src/Curiosity/curiosity.tsx]: Let's use the convention we've used else where especially on this page to call it top-page-break and same for the other ones

**Review by ChristianMcc — COMMENTED** [review]
couple of formatting comments :)

**Review by ChristianMcc — APPROVED** [review]

**Comment by chloejonessoftwire** [comment]
added extra styling

**Pull request merged**